electrum/lnchannel.py

```diff
@@ -1167,16 +1167,16 @@ def _verify_htlc_sig(self, *, htlc: UpdateAddHtlc, htlc_sig: bytes, htlc_directi
                                                           commit=ctx,
                                                           ctx_output_idx=ctx_output_idx,
                                                           htlc=htlc)
-        preimage_hex = htlc_tx.serialize_preimage(0)
-        pre_hash = sha256d(bfh(preimage_hex))
+        pre_hash = htlc_tx.serialize_preimage(0)
```

Comment:
that renaming is a bit confusing, `pre_hash` is not a hash anymore

Reply:
Oooooh! So that stood for "hash of preimage"?
I thought it was pre-hash, as in before-hashing.
That's why I renamed it, because with that reading it was a misnomer :D

Reply:
got it :-)